portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -23,14 +23,14 @@
   private final String name;
   private final String email;
   private final String message;
-  private final String timestamp;
+  private final String datetime;
 
   public Comment(long id, String name, String email, String message, long timestamp) {
     this.id = id;
     this.name = name;
     this.email = email;
     this.message = message;
-    this.timestamp = this.getTimestamp(timestamp);
+    this.datetime = this.getTimestamp(timestamp);
```

Comment:
Let's update the name of this function to match what it returns, as well.

Reply:
whoops, will update this!